src/Engine/Systems/Subsystems/PhysicsRenderBridge.hpp

```
@@ -0,0 +1,41 @@
+		std::lock_guard<std::mutex> lock(m_mutex);
+		m_readBuffer = std::move(frame);
+		m_frameReady = true;
+	}
+
+
+	/* Returns a copy of the latest snapshot.
+	   @param outFrame: The frame to be populated with the latest snapshot.
+	   
+	   Returns false if no new frame has been published since last consume, in which case outFrame is populated with the most recent old snapshot.
+	*/
+	inline bool consume(Buffer::PhysRendFramePacket &outFrame) {
```

Clang (14.0.6)

[warning] use a trailing return type for this function

(modernize-use-trailing-return-type)